tx_utils.py

```diff
@@ -0,0 +1,358 @@
+import json
```

Comment:
remove non-needed functions from this file

Reply:
removed.